src/Arduino_UnifiedStorage.cpp

```diff
@@ -0,0 +1,20 @@
+#include "Arduino_UnifiedStorage.h"
+#include "Utils.h"
+
+bool Arduino_UnifiedStorage::debuggingModeEnabled = false;
+
+void Arduino_UnifiedStorage::testPrint(String message){
```

Comment:
we need a separate method because if we would use the debugPrint method it would need the boolean debugging flag to be true, and it would polute the already long output of the test sketches with debug information, this way, we can use testPrint to simply print the output on RS485 or Serial, and debugPrint to include all the verbose output